{{cookiecutter.repo_name}}/scripts/git-setup.sh

```diff
@@ -20,4 +20,4 @@ git add . \
     echo "#################################################"
     echo "GIT SETUP COMPLETE! You're set up to develop.  Please note you're now on the dev branch"
     echo "#################################################"
-}
\ No newline at end of file
+}
```

Comment:
looks quite similar to scripts/init.sh - is there a need for both?
If they are different it would probably be worth pointing out the differences in the README

Reply:
Yeah, good shout.  They are different - the one sets up the local repo, the other (this one) links it with your remote on github.

Reply:
I read through the main README on cookiecutter again and it is explained there, if you take the heading/context into account. I believe that sufficiently explains the difference and what's happening.